ReelTalk/containers/MovieDetailView.js

```diff
@@ -93,6 +93,7 @@ var MovieDetailView = React.createClass({
 var styles = StyleSheet.create({
     scrollView: {
       flex: 1,
+      marginBottom: 50
```

Comment:
That is weird, the automaticallyAdjustContentInsets={true} is supposed to take care of this